From b6dc3237d467d5d08bc5ed459fcb9b40fb1a0593 Mon Sep 17 00:00:00 2001
From: Kyle Mestery <mestery@mestery.com>
Date: Tue, 7 Apr 2015 20:53:42 +0000
Subject: [PATCH] Fix broken unit tests for networking-odl

Commit 1105782e3914f601b8f4be64939816b1afe8fb54 on the Neutron side
reorganized unit tests. This fixes the location of test_ml2_plugin,
which is imported by the networking-odl unit tests code, and
consequently revives successful unit test runs for networking-odl.

Change-Id: I1b517b907c4bf88bdf89dc2532b6b3a24ff57309
---
 networking_odl/tests/unit/ml2/test_mechanism_odl.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/networking_odl/tests/unit/ml2/test_mechanism_odl.py
+++ b/networking_odl/tests/unit/ml2/test_mechanism_odl.py
@@ -28,7 +28,7 @@ from neutron.plugins.ml2 import driver_a
 from neutron.plugins.ml2.drivers.opendaylight import driver
 from neutron.plugins.ml2 import plugin
 from neutron.tests import base
-from neutron.tests.unit.ml2 import test_ml2_plugin as test_plugin
+from neutron.tests.unit.plugins.ml2 import test_plugin as test_plugin
 from neutron.tests.unit import testlib_api
 
 HOST = 'fake-host'
